src/Jira/Api.php

```diff
@@ -61,9 +60,9 @@ class Api
     protected $statuses;
 
     /**
-     * create a jira api client.
+     * Create a JIRA API client.
      *
-     * @param $endpoint
+     * @param string $endpoint URL
      * @param AuthenticationInterface $authentication
```

Comment:
As said in the PR its a step in the right direction I think, but in no way complete. Did it in the constructor of API since that variable might be vague for some new to the lib.